### ISSUE TITLE
fix important builders so we don't do model first programming

### ISSUE THREAD

**Issue by despiegk** [threefoldtech/jumpscaleX#155]
instructions in 
https://github.com/threefoldtech/jumpscaleX/blob/development/docs/flist/auto_generation.md

are very wrong

we need to use language = DSL for specifiying how to sandbox/build/...

please update instructions do how to do builders:
https://github.com/threefoldtech/jumpscaleX/blob/development/docs/Internals/Builders.md

### important builders prio 1

- python
- openssl
- lua/openresty/nginx
- basically everything required for our sandbox
- caddy
- etcd
- traeffik
- coredns

### important builders prio 2
- humhub relevant components

## branch

 use branch development_builders

**Comment by despiegk** [comment]
check fix #109

**Comment by ranatrk** [comment]
Implement sandbox functions for CoreDns and OpenResty: 
https://github.com/threefoldtech/jumpscaleX/commit/23131349b90a2241697c205f2dce1954e9f94498

**Comment by ahmadnazih** [comment]
Traefik and Caddy builders are finished with the creation of their sandbox and flist

**Comment by andrewayoub** [comment]
A document about how to create a new builder and an example added [here](https://github.com/threefoldtech/jumpscaleX/blob/development_builders/docs/howto/Create%20a%20new%20Builder.md)

**Comment by andrewayoub** [comment]
A pull request created [here](https://github.com/threefoldtech/jumpscaleX/pull/167) to track the progress

**Comment by ahmadnazih** [comment]
Humhub done - Startup toml file for nginx needs to be created 
https://github.com/threefoldtech/jumpscaleX/commit/ae1214b15d907cc815eb9ca83471bcc63a1820f6
